boardswarm-protocol/proto/boardswarm.proto

```diff
@@ -23,7 +23,7 @@ service Boardswarm {
   rpc VolumeIo (stream VolumeIoRequest) returns (stream VolumeIoReply);
 
   // Commit all data in an uploader specific manner; e.g. may trigger a USB reset for DFU devices
-  rpc VolumeCommit(VolumeRequest) returns (google.protobuf.Empty);
+  rpc VolumeCommit(VolumeCommitRequest) returns (google.protobuf.Empty);
```

Comment:
I really need to pin down the semantics for commit somewhat better; (and ditto shutdown wrt. the requests). As it's getting somewhat confusing.. But generally it was meant as finish-everything-the-volume is doing hence their on purpose not being a volume target.

For the `continue`  on fastboot this seems very much a case of committing in that sense as well as there is no specific target per se; It is meant to continue the normal boot flow. As you can also see from your next commit which ignores the target entirely when running continue :)

Reply:
> I really need to pin down the semantics for commit somewhat better; (and ditto shutdown wrt. the requests). As it's getting somewhat confusing.. But generally it was meant as finish-everything-the-volume is doing hence their on purpose not being a volume target.
> 
> For the `continue` on fastboot this seems very much a case of committing in that sense as well as there is no specific target per se; It is meant to continue the normal boot flow. As you can also see from your next commit which ignores the target entirely when running continue :)

Fastboot in the device ignores the target, but the `commit` behavior is different if the target volume is the special `stage` or any other target.